Framework/Core/test/test_Graphviz.cxx

```diff
@@ -30,11 +30,20 @@ void lineByLineComparision(const std::string& as, const std::string& bs)
 
   char bufferA[1024];
   char bufferB[1024];
-
+ 
+  bool same{true};
   while (a.good() && b.good()) {
     a.getline(bufferA, 1024);
     b.getline(bufferB, 1024);
     BOOST_CHECK_EQUAL(std::string(bufferA), std::string(bufferB));
+    if (std::string(bufferA) != std::string(bufferB)) {
+      same=false;
+    }
+  }
+  if (!same){
+  std::cout << as << "\n";
+  std::cout << "\n";
+  std::cout << bs << "\n";
```

Comment:
Is this e temporary debug output which should be removed again?

Reply:
You're perfectly right. Sorry about that. Fixed.